### ISSUE TITLE
For 0.6.6 skins "keep it simple"

### ISSUE THREAD

**Issue by felisucoibi** [RetroShare/RetroShare#1616]
Only one white skin, only one dark style (qdark) , make them to look well.
Remove the rest.

Now all the designs are more or less buggy, colors etc...

**SKINS THAT NEEDS TO BE REMOVED OR FIXED:**
BLACKNIGHT : SAME COLOR THAN BACKGROUND AND OTHER PROBLEMS
BLUE: WIHTE LETTER IN VERY WIHTE BACKGROUND, NOT POSSIBE TO READ
GROOVE: UGLY NEEDS SOME LOVE AND COLORS
ORANGE: UGLY, NEEDS SOME LOVE IN BACKGROUNDS
QLIVE : UGLY NEEDS SOME LOVE IN BACKGROUNDSOVE IN
REDSCORIPON: UBLY NEEDS SOME LOVE IN BACKGROUNDS
SYLVERGREY: BROKEN NEEDS FIXES
UUS: UGLY NEEDS FIXES IN BACKGROUNDS
YABA HIPERUGLY
YEAH BROKEN.

Fix the blue link problem of qdark and that's all 2 options, but two working options.

**Comment by csoler** [comment]
I fully support this.

**Comment by felisucoibi** [comment]
If nobody else has anything to say if its posible fix the blue default link of qdarkstyle and renmove the rest of Skins. i think @defnax updated And fixed qdarkstyle 2 months ago only missing the Fix dor defsult Link color. I tired to Fix the blue link problem it but i Got lost inside qt css

**Comment by tiotrom** [comment]
I fully agree with this! I've seen that people are reluctant to use RetroShare because is "ugly". I know, shallow values, and I don't share them, but I hope a modern looking RetroShare will attract more people. We have a custom linux distro based on manjaro and will include retroshare by default. It is such a great tool!

**Comment by tiotrom** [comment]
Btw when do you plan the next release? Cheers!

**Comment by felisucoibi** [comment]
**SKINS THAT NEEDS TO BE REMOVED OR FIXED:**
BLACKNIGHT : SAME COLOR THAN BACKGROUND AND OTHER PROBLEMS
BLUE: WIHTE LETTER IN VERY WIHTE BACKGROUND, NOT POSSIBE TO READ
GROOVE: UGLY NEEDS SOME LOVE AND COLORS
ORANGE: UGLY, NEEDS SOME LOVE IN BACKGROUNDS
QLIVE : UGLY NEEDS SOME LOVE IN BACKGROUNDSOVE IN
REDSCORIPON: UBLY NEEDS SOME LOVE IN BACKGROUNDS
SYLVERGREY: BROKEN NEEDS FIXES
UUS: UGLY NEEDS FIXES IN BACKGROUNDS
YABA HIPERUGLY
YEAH BROKEN.

**Comment by defnax** [comment]
@csoler @thunder2 close
